app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -44,7 +44,7 @@ class EventsFragment : Fragment() {
                 val bundle = Bundle()
                 bundle.putLong(fragment.EVENT_ID, eventID)
                 fragment.arguments = bundle
-                eventsViewModel.loadEventsAgain = true
+                eventsViewModel.loadEventsWithAnimation = false // No slide up animation
```

Comment:
Explain why it is being set to false here

Reply:
if you open the event details fragment and return to the events fragments the UI flickers.
I am using this boolean for that particular case

Reply:
I meant explain this in comment. Your comment here is obvious. Comments are used for non-obvious things

Reply:
comment removed